OloEngine/src/OloEngine/Renderer/Debug/CapturedFrameData.cpp

```
@@ -0,0 +1,49 @@
+#include "OloEnginePCH.h"
+#include "CapturedFrameData.h"
+
+namespace OloEngine
+{
+    const char* CapturedCommandData::GetCommandTypeString() const
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)